plugins/pulp_rpm/plugins/serializers.py

```diff
@@ -49,6 +50,20 @@ class Meta:
         remapped_fields = {'errata_from': 'from',
                            'errata_id': 'id'}
 
+    def serialize(self, unit):
+        """
+        Convert a single unit to it's dictionary form.
+
+        Add to errratum unit its pkglist.
+
+        :param unit: The object to be converted
+        :type unit: object
+        """
+        from pulp_rpm.plugins.db.models import ErratumPkglist
```

Comment:
I do not like this one but serializer was a good spot to add pkglists back not to change API (response will still contain pkglists in this case). Other ideas?